tools/tf2openapi/tfmodel/node.go

```diff
@@ -0,0 +1,5 @@
+package tfmodel
+
+type Node interface {
```

Comment:
You've talked a lot about this "Visitor" pattern. Can you link somewhere to this strategy you're following?